src/main/java/com/app/katchup/Meeting/MeetingController.java

```diff
@@ -0,0 +1,22 @@
+package com.app.katchup.Meeting;
+
+import com.app.katchup.Meeting.model.Meeting;
+import com.app.katchup.Meeting.model.Poll;
+import com.app.katchup.MeetingResponse.model.Decision;
+import com.app.katchup.MeetingResponse.model.MeetingResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class MeetingController {
+    @Autowired
+    MeetingService meetingService;
+
+    @PostMapping("/meeting/create")
+    public ResponseEntity<Meeting> postMeeting(@RequestBody Meeting meeting){
+        Meeting meetingCreated=meetingService.createMeeting(meeting);
+        return new ResponseEntity<>(meetingCreated, HttpStatus.CREATED);
```

Comment:
please add null check for the returned object, if it is null, HttpStatus.Conflict(409)
else created